app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt

```diff
@@ -64,14 +63,21 @@ fun EventInfoSheet(
     val hour = event.startDate.hour
     val minute = event.startDate.minute
     val displayMonth = if (month < 10) "0$month" else month.toString()
-    val displayDate = "$day/$displayMonth\n$hour:$minute"
+    val displayMinute = if (minute < 10) "0$minute" else minute.toString()
+    val displayDay = if (day < 10) "0$day" else day.toString()
+    val displayHour = if (hour < 10) "0$hour" else hour.toString()
+    val displayDate = "$displayDay/$displayMonth\n$displayHour:$displayMinute"
```

Comment:
Instead of doing this by hand you could use a `DateTimeFormatter` as follows: `val formatter = DateTimeFormatter.ofPattern("dd/MM\nHH:mm")` and then simply `val displayDate = formatter.format(event.startDate)`

Reply:
oh that's cool I'll use that